satellite/referral/offersweb/static/css/style.css

```diff
@@ -0,0 +1,9 @@
+html{
+    height:100vh;
+    width:100vw;
+}
+
+body{
+    height:100%;
+    width:100%;
+}
```

Comment:
Would be nice if we can take this out and use the exisiting CSS data..

Reply:
@stefanbenten The existing CSS data is scoped and won't apply to us. The look and feel will ultimately be up to DJ so as long as we style it the way he wants, having separate CSS files shouldn't be a problem. See https://github.com/storj/storj/pull/1938/files#r283005800